NewClass.java

```diff
@@ -0,0 +1,8 @@
+class NewClassdhjw{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/app/gagsdeep/TestSonar/pullRequest?prid=4664199)